Assets/Scripts/Controllers/WorldController.cs

```diff
@@ -131,6 +131,7 @@ public void Start()
         spawnInventoryController.SetUIVisibility(Settings.GetSetting("DialogBoxSettings_developerModeToggle", false));
 
         cameraController.Initialize();
+        cameraController.Moved += this.World.OnCameraMoved;
 
```

Comment:
Me too, but when I tried, it seems that world is created before cameracontroller, so I'm not sure how